internal/providers/terraform/ibm/testdata/is_instance_test/is_instance_test.golden

```diff
@@ -2,17 +2,17 @@
  Name                                                  Monthly Qty  Unit                        Monthly Cost 
                                                                                                              
  ibm_is_instance.testBalancedInstance                                                                        
- ├─ CPU hours (32 CPUs, us-east-1)                          23,360  CPU hours                        $619.23 
- ├─ Memory hours (128 GB, us-east-1)                        93,440  Memory hours                     $394.52 
+ ├─ CPU hours (32 CPUs, us-east-1)                     21,072.0000  CPU hours                        $558.58 
```

Comment:
why did the quantities change? they shouldn't be affected by the multiplier

Reply:
Because I multiply the quantities, there are no "price" multipliers